<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<node name="tb_distancecloud2d_node"  pkg="tb_distancecloud" type="tb_distancecloud2d_node" respawn="true" output="screen">
		<param name="update_area_z_radius" 		 	 type="double" value="1.5" />
		<param name="collision_radius" 		 	 type="double" value="15" />
		<param name="normalize_RGB" 		 	 type="bool" value="true" />
	</node>
</launch>
